tf2onnx/tfonnx.py

```diff
@@ -43,7 +43,8 @@
 TARGET_RS5 = "rs5"
 TARGET_RS6 = "rs6"
 TARGET_CAFFE2 = "caffe2"
-POSSIBLE_TARGETS = [TARGET_RS4, TARGET_RS5, TARGET_RS6, TARGET_CAFFE2]
+TARGET_ONNXRUNTIME = "onnxruntime"
```

Comment:
I'm pretty sure this will be fixed in some future version - kind of saying there should be a version in the target similar like rs4, rs5, rs6 (I did this wrong for caffe2).
The current version of onnxruntime should have the same issues as rs6 ... maybe use rs6 in this case ?

Reply:
ok, sure, will use rs6.